blog/executor-plugins.md

```diff
@@ -0,0 +1,340 @@
+
+helm install --version v0.17.0 botkube --namespace botkube --create-namespace \
+--set communications.default-group.socketSlack.enabled=true \
+--set communications.default-group.socketSlack.channels.default.name=${SLACK_CHANNEL_NAME} \
+--set communications.default-group.socketSlack.channels.default.bindings.executors=['plugin-based'] \
+--set communications.default-group.socketSlack.appToken=${SLACK_APP_TOKEN} \
+--set communications.default-group.socketSlack.botToken=${SLACK_BOT_TOKEN} \
+# example configuration for `gh` executor
+-f https://gist.github.com/mszostok/88e6852f93429928d1183e52226921fc/raw \
+--set 'plugins.repositories.botkube-plugins.url'=${PLUGINS_URL} \
+--set 'executors.plugin-based.botkube-plugins/gh.config.github.repository'=${REPOSITORY} \
+--set 'executors.plugin-based.botkube-plugins/gh.config.github.token'=${GITHUB_TOKEN} \
+botkube/botkube
+```
+
+## Summary
+
+Botkube executors are powerful because they can glue together three important parts: Kubernetes clusters, communicators, and tools of your choice. There would be nothing special about it if it wasn't, in fact, unburdening you of those implementation-specific details.
+
+As you noticed, you can focus purely on your business logic. Without the need to use different chat libraries, know how to establish secure connection, or make your extension available only on specific channels. What's more, not only do you not have to learn it, but you don't have to support it either, as we do it for you.
+
+Once Botkube is deployed, your extension will be available to you and your teammates in a given channel. There is no need to maintain your local setup. Thanks to that, you can also easily run executors on private clusters.
+
+Botkube extensions can be used with other Botkube functionality too. It means that you can use them to create automation. We will shed more light on that in the next blog post. Stay tuned!
+
+> Implement once, access everywhere (Slack, Discord, Mattermost, MS Teams).
+
+## How can I get involved?
+
+The implemented plugin is as simple as possible. However, it is a great base for further extension based on your needs; for example: introduce your own Kubernetes annotation to route the notification to a specific repository, add a threshold to create issues only for constantly failing Pods, etc. The possibilities are endless, and we cannot wait to see what kind of great workflows you will create!
+
+As always, we want to hear your feedback and ideas about Botkube. Help us plan the Botkube roadmap, get the features you’d like implemented.
+
+There are plenty of options to contact us:
+
+- [GitHub issues](https://github.com/kubeshop/botkube/issues)
+- [Slack](https://join.botkube.io/)
+- or email our Product Leader at [blair@kubeshop.io](mailto:blair@kubeshop.io).
+
+Thank you for taking the time to learn about Botkube 🙌
+
+### Resources
+
+- The GitHub repository with the `gh` executor [source-code](https://github.com/kubeshop/botkube-plugins-template)
```

Comment:
it will be updated to point to a given file once the https://github.com/kubeshop/botkube-plugins-template/pull/7 will be merged.